Automattic/wp-calypso
# Extracts Notice logic into the JetpackDisconnectedNotice component

#### Proposed Changes

* Extracts the state management logic for the site disconnect section.
* Adds it into `JetpackDisconnectedNotice `.

#### Testing Instructions



* Apply D86035-code.
* Navigate to the plugins marketplace.
* The disconnected site notice should appear.

#### Pre-merge Checklist






Fixes #64546
